fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/home/HomeActivityTest.java

```diff
@@ -48,28 +47,6 @@ public void createButtonBringsToCreateScreen() {
     fragmentContainer().check(matches(withChild(withId(createFragmentId()))));
   }
 
-  @Test
-  public void logOutMenuTest() {
-    initializeWallet(activityScenarioRule);
-    openActionBarOverflowOrOptionsMenu(
-        InstrumentationRegistry.getInstrumentation().getTargetContext());
-    walletLogOutMenuItem().perform(click());
-    assertThat(dialogPositiveButton(), allOf(withText("CONFIRM"), isDisplayed()));
-    assertThat(dialogNegativeButton(), allOf(withText("CANCEL"), isDisplayed()));
-    dialogPositiveButton().performClick();
-    fragmentContainer().check(matches(withChild(withId(walletId()))));
-  }
-
-  @Test
-  public void clearDataTest() {
-    initializeWallet(activityScenarioRule);
-    openActionBarOverflowOrOptionsMenu(
-        InstrumentationRegistry.getInstrumentation().getTargetContext());
-    clearDataMenuItem().perform(click());
-    assertThat(dialogPositiveButton(), allOf(withText("YES"), isDisplayed()));
-    assertThat(dialogNegativeButton(), allOf(withText("NO"), isDisplayed()));
-  }
-
```

Comment:
Unfortunately with the new ToolBar component I was not able to find a way to click on the menu icon (the 3 dots), I could not find any id or contentDescription that could have helped doing so. I also found no resources online for doing that for that component

Reply:
Wouldn't it be possible to re-add it manually ?

Anyway, I don't think this is a big issue right now.
We will re-add the settings button later if needed.

Reply:
Well it's embarassing but I just thought about the way it was done in Karate tests which is to use the content description "More options", I tried it... and it works

Reply:
Oh well